examples/tutorials/vectors.py

```diff
@@ -0,0 +1,459 @@
+"""
+Plot vectors
+==========
+
+Plotting vectors is handled by :meth:`pygmt.Figure.plot`.
+
+.. note::
+
+    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+    To see the figures while using a Python script instead, use
+    ``fig.show(method="external")`` to display the figure in the default PDF viewer.
+
+    To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
+    is the desired name and file extension for the saved figure.
+"""
+# sphinx_gallery_thumbnail_number = 6
+
+import numpy as np
+import pygmt
+
+########################################################################################
+# Plot Caretesian Vectors
+# ----------
+#
+# Create a simple Cartesian vector using a starting point through
+# ``x``, ``y``, and ``direction`` parameters. The direction is specified
+# by a list of two 1d arrays structured as ``[[angle_in_degrees], [length]]``
+#
+# On the shown figure, the plot is projected on a 10cm X 10cm region,
+# which is specified by the ``projection`` parameter.
+# The magnitude of the vector also uses centimeters by default but
```

Comment:
I'd like to suggest using "length" instead of "magnitude" in this tutorial because it is more intuitive, isn't it? Meanwhile, it is consistent with the `direction` parameter.
```suggestion
# The length of the vector also uses centimeters by default but
```